commons/src/test/java/org/hyades/common/KafkaTopicTest.java

```diff
@@ -1,21 +1,17 @@
 package org.hyades.common;
 
 import io.quarkus.test.junit.QuarkusTest;
-import org.hyades.config.KafkaTopicConfig;
 import org.junit.jupiter.api.Test;
 
-import javax.inject.Inject;
-
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 @QuarkusTest
 public class KafkaTopicTest {
 
-    @Inject
-    KafkaTopicConfig kafkaTopicConfig;
-
     @Test
     void testKafkaTopicConfig() {
         assertEquals("dtrack.vulnerability.mirror.osv", KafkaTopic.MIRROR_OSV.getName());
+        System.setProperty("api.topic.prefix", "customPrefix.");
```

Comment:
The property needs to be unset again after the test, otherwise other tests will be impacted.

Another option would be to use `@TestProfile`, you can find multiple usages of it throughout the code base if you need examples.